### DISCUSSION TITLE
Missing results

### DISCUSSION THREAD

**Discussion by ibash** [community/community#43891] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Steps:
1. Go to this repo: https://github.com/th33xitus/kiauh
2. Search for "install_klipper_packages" in the repo

Expected:
Finds two results:
- https://github.com/th33xitus/kiauh/blob/8493269c6fc677092e6b70bfb29dd71b004b6e2c/scripts/klipper.sh#L275
- https://github.com/th33xitus/kiauh/blob/f70faa52cc13045c8d8e424a233fd4a26bac3c7a/scripts/klipper.sh#L204

Actual:
Only the result on line 275 is found, not on line 204.
Using the old code search does find both results.

This is a problem because if results are missing then it's not possible to trust what code search shows. It's _okay_ if code search is noisy as long as it's correct.

**Comment by look** [comment]
Thanks for the feedback @ibash. The new code search prioritizes the "best" result in the file, in this case the function definition on line 275. The result view indicates that there are more results in the file, which you can see if you click on it.

This is something we can improve though, so thanks for raising the problem.
